app-2/src/main/java/tech/relaycorp/letro/onboarding/registration/parser/RegistrationMessageParser.kt

```diff
@@ -0,0 +1,22 @@
+package tech.relaycorp.letro.onboarding.registration.parser
+
+import tech.relaycorp.letro.awala.message.Message
+import tech.relaycorp.letro.onboarding.registration.dto.RegistrationResponse
+import java.nio.charset.Charset
+import javax.inject.Inject
+
+interface RegistrationMessageParser {
```

Comment:
BTW I extracted parsing logic for registration response here, so you will be able to do your stuff here as we discussed today

Reply:
Thanks! This is very helpful